concordium-sdk/src/main/java/com/concordium/sdk/responses/identityproviders/IpInfo.java

```diff
@@ -0,0 +1,43 @@
+package com.concordium.sdk.responses.identityproviders;
+
+import com.concordium.sdk.crypto.ed25519.ED25519PublicKey;
+import com.concordium.sdk.crypto.pointchevalsanders.PSPublicKey;
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.types.UInt32;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+import lombok.Data;
+import lombok.ToString;
+
+@Data
+@ToString
+public class IpInfo {
```

Comment:
There is already defined `IdentityProviderInfo` in the SDK.

Reply:
done, used